draft-ietf-mls-protocol.md

```diff
@@ -1183,13 +1185,13 @@ A group member generates this message by requesting a UserInitKey
 from the directory for the user to be added, and encoding it into an
 Add message.
 
-The new participant processes Welcome and Add messages together as
-follows:
+The client about to join the group processes Welcome and Add
```

Comment:
I might rephrase this as "The client joining the group..."

Reply:
Fixed.